spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
 require "simplecov"
-SimpleCov.start { command_name "spec" }
```

Comment:
I thought we needed these to prevent one test suite from overwriting all of the data from the previous one

Reply:
Perhaps in the past, but the current version of SimpleCov automatically creates two sections named "RSpec" and "Unit tests" for rpsec and minitest respectively.  This keeps the defaults, and then SimpleCov automatically merges them together into the JSON payload when it sees `CC_TEST_REPORTER_ID` as an env var.